### ISSUE TITLE
How can I install blender2faust?

### ISSUE THREAD

**Issue by FinishedBz1** [grame-cncm/faust#897]
In several documentations about the physical modeling toolkit there was a mesh2faust add-on for blender mentioned.
Unfortunately I cannot find it.

**Comment by rmichon** [comment]
Indeed, Smilen Dimitrov wrote this a couple of years ago: https://ccrma.stanford.edu/~rmichon/publications/doc/SMC-18-BF.pdf
but just like you, I can't find the actual tool anywhere. I haven't talked to Smilen in a long time and the current contact info I have is not up to date :(. mesh2faust should still work though :)

**Comment by thekartikeyamishra** [comment]
### According to me these steps should help you out 

> It's possible that the add-on is no longer maintained or has been renamed. However, you can try searching for it on GitHub or Blender's add-on repositories. If you still can't find it, you might need to look for alternative solutions or create your own script to achieve the desired functionality.

1.  Download the add-on (usually a .zip file) to your computer.
2.  Open Blender and go to Edit > Preferences.
3.  In the Preferences window, click on the "Add-ons" tab.
4.  Click on the "Install" button at the top right corner.
5.  Navigate to the downloaded .zip file, select it, and click "Install Add-on".
6.  Once installed, you should see the add-on in the list. Enable it by checking the checkbox next to the add-on's name.

After installing and enabling the add-on, you should be able to access its features within Blender. The location of the add-on's features may vary depending on the add-on itself. Check the add-on's documentation for specific instructions on how to use it.

> If you're unable to find the mesh2faust add-on or an alternative, you might need to consider other approaches to achieve your goal. You can try looking into Blender's built-in features or other add-ons that might help with physical modeling. Additionally, you can explore the possibility of writing your own Python script to create the desired functionality within Blender.
